modules/social_features/social_profile/modules/social_profile_organization_tag/social_profile_organization_tag.install

```diff
@@ -157,3 +159,44 @@ function social_profile_organization_tag_update_130001(): void {
   user_role_grant_permissions('contentmanager', ['create terms in profile_organization_tag']);
   user_role_grant_permissions('sitemanager', ['create terms in profile_organization_tag']);
 }
+
+/**
+ * Create entity-form-display or hide path field from Organization Tag taxonomy.
+ */
+function social_profile_organization_tag_update_130002(): void {
+  $storage = \Drupal::entityTypeManager()->getStorage('entity_form_display');
+  $form_display = $storage->load('taxonomy_term.profile_organization_tag.default');
+
+  // When entity-form-display isn't be created,
+  // it'll create from install folder.
```

Comment:
Please update english here and where you copy pasted it :)
`// When entity-form-display isn't created,`
`// It will be created from the install folder.`